arch/arm/core/aarch64/Kconfig

```diff
@@ -58,6 +58,12 @@ config IPM_CONSOLE_STACK_SIZE
 
 if CPU_CORTEX_A
 
+config ARMV8_A_NS
+	bool "ARMv8-A Normal World (Non-Secure world of Trustzone)"
+	help
+	  This option signifies Zephyr that is running in Trustzone
```

Comment:
s/signifies Zephyr that/signifies that zephyr/

Reply:
Done